### DISCUSSION TITLE
Allow a tolerance of a few mistakes before stopping the test with the minimum accuracy option

### DISCUSSION THREAD

**Discussion by userrand** [monkeytypegame/monkeytype#4522] [Ideas] [answered]
Currently with the option to stop the test if it reaches below a minimum accuracy, the test can fail quite early if just one mistake is made at the beginning. This can be a bit annoying. It would be nice if there was a setting where the test only stops if the user has typed a certain number of characters.

**Comment by Miodec** [comment, marked as answer]
It used to ignore a few starting words but then people found it confusing that either:
 - the test didnt stop when they made mistakes
 - they made mistakes but then started typing correctly and it still failed because they crossed the "free words" threshold but their accuracy was still below the limit

Allowing some words just delays the test fail, so that got removed.

**Reply by userrand** [reply]
Thank you for the explanation.
